src/katgpucbf/fgpu/engine.py

```diff
@@ -1581,21 +1590,13 @@ def comma_string_to_float(comma_string: str) -> tuple[float, float]:
             phase, phase_rate = comma_string_to_float(phase_str)
 
             delay_samples = delay * self.adc_sample_rate
-            # For compatibility with MeerKAT, the phase given is the net change in
-            # phase for the centre frequency, including delay, and we need to
-            # compensate for the effect of the delay at that frequency. The centre
-            # frequency is 4 samples per cycle, so each sample of delay reduces
-            # phase by pi/2 radians.
-            delay_phase_correction = 0.5 * np.pi * delay_samples
-            phase += delay_phase_correction
-            phase_rate_correction = 0.5 * np.pi * delay_rate
             new_linear_models.append(
                 LinearDelayModel(
                     start_sample_count,
                     delay_samples,
                     delay_rate,
                     phase,
-                    phase_rate / self.adc_sample_rate + phase_rate_correction,
```

Comment:
Not that I'm emotionally invested in it, but what happened to `phase_rate_correction`?
* Is it being applied/incorporated somehow elsewhere?
* Or is the updated approach of delay handling/calculation handling this correction factor?

Reply:
We're now handling phase and phase_rate with the same interpretation as they're given i.e. the total phase adjustment at the centre of the band, instead of at channel 0. So this fiddling is no longer needed.